README.md

```diff
@@ -282,7 +282,7 @@ Creates an application pool in IIS.
 
 ##### Root Items
 
-- `pool_name` - name attribute. Specifies the name of the pool to create.
+- `name` - name attribute. Specifies the name of the pool to create.
```

Comment:
Changing this from pool_name to name has caused us some issues when upgrading to this version of the cookbook.  We use our resource name attribute as more of a description of what the resource is doing, so all of our notifies used that description to reference them.  Since the name of the resource is now over-ridden by this change we have to reference the resource in the resource collection by the name of our iis pool, which has cause conflicts and some messy recipe design.  This is also different from the other resources like iis_site and iis_app that have site_name and application_name attributes instead of overriding the name attribute.

Reply:
@nwesoccer we should of bumped the version to 7 as to depict that this wasn't a backwards compatible update.  Our bad, all resources will be moving to `name` as it is more appropriate to the self idea..